docs/sanitizers/error-alloc-dealloc-mismatch.md

```diff
@@ -1,64 +1,67 @@
 ---
 title: "Error: alloc-dealloc-mismatch"
-description: "Source examples and live debug screenshots for alloc-dealloc-mismatch errors."
-ms.date: 03/02/2021
+description: "Learn about the alloc-dealloc-mismatch Address Sanitizer error."
```

Comment:
Is this the preferred description? If so, I can change all the others to maintain uniformity.